consensus/core/src/authority_node.rs

```diff
@@ -104,13 +106,15 @@ mod tests {
         let block_verifier = TestBlockVerifier {};
 
         let (own_index, _) = committee.authorities().last().unwrap();
+        let block_signer = NetworkKeyPair::from_bytes(keypairs[0].0.as_bytes()).unwrap();
```

Comment:
we will use the block signer for everything right in the consensus protocol? the signer below for now has no plans to be used?

Reply:
Basically yes. From the so far discussions it also seems that the random beacon will get implemented out side of consensus , so probably this won't be used. I can do a clean up on follow up PR.